SpatialGDK/Source/SpatialGDK/Public/Interop/AbstractStaticComponentView.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/AuthorityIntent.h"
+#include "SpatialConstants.h"
+#include "WorkerSDK/improbable/c_worker.h"
+
+#include "UObject/Object.h"
+
+#include "AbstractStaticComponentView.generated.h"
+
+UCLASS()
+class SPATIALGDK_API UAbstractStaticComponentView : public UObject
```

Comment:
Do StaticComponetViews still have to be derived from a `UObject`?

Reply:
They are currently, and I didn't think to change that in this PR.

Reply:
It's that since the class' hierarchies change in this PR, I thought it's fine to also remove a UObject dependency. I agree it could be a different PR, so wouldn't block on this.

Reply:
I think we should definitely do that, but I agree not in this PR.